plugin/src/main/java/app/cash/paraphrase/plugin/ParaphrasePlugin.kt

```diff
@@ -49,12 +50,28 @@ class ParaphrasePlugin : Plugin<Project> {
 
   private fun Project.addDependencies() {
     val isInternal = properties["app.cash.paraphrase.internal"].toString() == "true"
+
+    // Automatically add the runtime dependency.
     val runtimeDependency: Any = if (isInternal) {
       dependencies.project(mapOf("path" to ":runtime"))
     } else {
       "app.cash.paraphrase:paraphrase-runtime:${BuildConfig.VERSION}"
     }
     dependencies.add("api", runtimeDependency)
+
+    // Automatically add the runtime Compose UI dependency if the Compose build feature is present.
+    afterEvaluate {
```

Comment:
When I tested this out in the sample project, the `android` extension hadn't been evaluated yet when Paraphrase is configured. Is this the right way to defer adding the dependency, or is there a more targeted way to do it specifically after the `android` extension is evaluated?

Reply:
Yeah this looks right. After evaluate runs after the build script has been evaluated at which point it should be safe to read these values.